button/button.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="button.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+        integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>RangerUI | docs</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <span class="navbar-heading"><a href="" class="ranger-ui  ranger-ui-name">RangerUI</a></span>
+        <section class="navbar-social-links">
+            <button class="navbar-btn">
+                <a href="https://github.com/Bhakti199" target="_blank" rel="noopener noreferrer"
+                    class="navbar-btn-link"><i class="fab fa-github"></i></a>
+            </button>
+            <button class="navbar-btn">
+                <a href="https://twitter.com/Bhaktee_" target="_blank" rel="noopener noreferrer"
+                    class="navbar-btn-link"><i class="fab fa-twitter"></i></a>
+            </button>
+            <button class="navbar-btn">
+                <a href="https://www.linkedin.com/in/bhakti-kharatkar-33178a200/" target="_blank" rel="noopener noreferrer"
+                    class="navbar-btn-link"><i class="fab fa-linkedin-in"></i></a>
+            </button>
+        </section>
+    </nav>
+
+    <ul class="sidebar" id="sidebarMenu">
+        <li class="sidebar-list-item">
+            <a href="../documentation.html" class="sidebar-link">Installation</a>
+        </li>
+        <li class="sidebar-list-item">
+            <a href="" class="sidebar-link">
+                Avatar
+            </a>
+        </li>
+        <li class="sidebar-list-item">
+            <a href="/alert/alert.html" class="sidebar-link">
+                Alert
+            </a>
+        </li>
+        <li class="sidebar-list-item">
+            <a href="" class="sidebar-link">
+                Badge
+            </a>
+        </li>
+        <li class="sidebar-list-item">
+            <a href="/button/button.html" class="sidebar-link">
+                Button
+            </a>
+        </li>
+        <li class="sidebar-list-item">
+            <a href="" class="sidebar-link">
+                Cards
```

Comment:
same as above

Reply:
I didn't get you here. Can you explain

Reply:
use Card here..as other components are mentioned

Reply:
ok got it.